### ISSUE TITLE
College students should base their choice of a field of study on the availability of jobs in that field. Write a response in which you discuss the extent to which you agree or disagree with the recommendation and explain your reasoning for the position you take. In developing and supporting your position, describe specific circumstances in which adopting the recommendation would or would not be advantageous and explain how these examples shape your position.

### ISSUE THREAD

**Issue by amardip11** [hemanth-ks97/GreAwa_TaskGenerator#6]
"Imagination is more important than knowledge", in this context , it was thought by experts of the world that, how important knowledge is. Albert Einstein above saying immerged us to the pool of  knowledge and wisdom -- to built our character in every field of study. It was good that the most of the college student would  always thought of in the deep of knowledge, rather than seeking the job in other field than that of the study of.   My intense choice is to be on the top of the field where I have invest my most of the time form childhood. 

On the same ways, there were many famous personality , who have deep intension in their own, no matter what the field was ,say, Albert Einstein was one of famous genius, who made several contribution on their won field, no matter what the condition he tolerate in the lifetime. He change the ways of thinking, on our own surrounding how one object related with other by  bringing in light of the theory of Relativity for the first time.

Moreover, it leads every people to on their own way, where most of the  inner frequency of person match with the outer ones. It leads more production of any thing in the company where they can produce hug outcome although by applying minimum efforts. However, the pocket of resistance , who were come to any field other than they surrounding with while in university, those of most have to face a huge hindrance of the way, full of pebble some of them were small, although company offering high facility and so on.

While, you contribute your most of the time and effort in your favor field, you don't have to work a single days, your passion work and it be game on the thrones. Others have to efforts on the tedious works only for material gains and so on. Several advantage we observe around us from  600 BC to till that passions in any field holds the total attention of the man although he have to drink Hemlock, ( poison drink by philosopher Socrates  to lead that wisdom be on the top of society) as same way great  mathematician Galileo  was edger on their own will to stand on truth . If those people believe the passion of any subject is nothing than now our society have some fault , that affect built our society soo sophisticated now.

Some of them who spend their time on the other field than that he studies he almost miss opportunity to be  universal form on expect. It was good to that the some of the students choose other field that what he or she study in school. 

In conclusion, most of the famous people of the world always eager on their own on, to make a great contribution for worlds, although what amount of time he have to invest no matter.

**Comment by amardip11** [comment]
can you please comment
